core/src/components/modal/test/animations/modal.e2e.ts

```diff
@@ -0,0 +1,40 @@
+import { expect } from '@playwright/test';
+import { configs, test } from '@utils/test/playwright';
+
+configs({ modes: ['ios'], directions: ['ltr'] }).forEach(({ config, title }) => {
+  test.describe(title('modal: animations'), () => {
+    test.beforeEach(async ({ page }) => {
+      await page.setContent(
+        `
+        <ion-modal is-open="true" trigger="open-modal"></ion-modal>
+      `,
+        config
+      );
+    });
+    test('card modal should clean up animations on dismiss', async ({ page }) => {
+      const ionModalDidDismiss = await page.spyOnEvent('ionModalDidDismiss');
```

Comment:
Add an annotation that keeps track of what GH issue this test is linked it.

```ts
testInfo.annotations.push({
  type: 'issue',
  description: 'https://github.com/ionic-team/ionic-framework/issues/28352',
});
```

Reply:
thank you! i always forget this is a thing. added in 26fb033434